expertise/create_dataset.py

```diff
@@ -71,16 +89,26 @@ def get_publications(self, author_id):
                 if not 'title' in publication.content or not publication.content.get('title'):
                     continue
             # Exclude blind Notes
-            if getattr(publication, 'original') is not None:
+            if hasattr(publication, 'original') and getattr(publication, 'original') is not None:
```

Comment:
can be shortened to `getattr(publication, 'original', None)`

Reply:
I didn't know you could do that!